_posts/2021-03-06-newpipe-0.20.11-released.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "NewPipe 0.20.11 released: Hotfixing YouTube and SoundCloud"
+short: "NewPipe 0.20.10 released"
+date: 2021-03-06 11:00:00 + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Welcome to the first hotfix release of 2021 - let us hope it will be the last one. This release fixes SoundCloud, receiving some YouTube content, an app crash and brings you a new translation.
+
+<!-- more -->
+
+## Changes
+
+A short release, so this blog post is short as well. These are the fixes which found their way into this release:
+
+* Thanks to [@TiA4f8R](https://github.com/TiA4f8R), getting related videos from channels works again in YouTube. [@XiangRongLin](https://github.com/XiangRongLin) getting those for playlists works as well.
+* [@golfinq](https://github.com/golfinq) was able to bring back comments on YouTube, so you are able to enjoy reading what other people think.
+* SoundCloud was completely broken, and now it is not. [@TobiGr](https://github.com/TobiGr) changed how NewPipe extracts the client ID from its website, which it then uses to extract more content from SoundCloud. [@B0pol](https://github.com/B0pol) fixed geo-restricted content, which are for example the trending lists or the top 50 songs.
+* In NewPipe itself was a crash when you tried to reorder channel groups. [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) hunted it down and fixed this crash.
+
+Next to these fixes, two new features are part of this app now. NewPipe now correctly resolves custom subpaths from YouTube like <code>/watch</code> or <code>/w/</code>. Northern Kudish speakers can enjoy NewPipe in their language.
+
+## Where to get this brand-new version
+
+NewPipe notifies you about new versions. You can download them when you press the notification, which will take you to the [GitHub Releases page](https://github.com/TeamNewPipe/NewPipe/releases).
+
+If you use the [F-Droid app](https://f-droid.org/), it, too, notifies you about updates for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/).
+
+!!! Please be aware that F-Droid is currently unable to publish a new version of NewPipe (the latest version available is 0.20.1, in which YouTube is already broken) due to an issue that applies only to reproducible apps. Hence it is our recommendation that you install NewPipe:
```

Comment:
What you said is now outdated, you can now say that the last version is available on F-Droid's official repo.

Reply:
huh, since the blogpost isn't released, thats right